tests/test_associationUtils.py

```diff
@@ -0,0 +1,63 @@
+# This file is part of pipe_tasks.
+
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import healpy as hp
+import numpy as np
+import unittest
+
+from lsst.pipe.tasks.associationUtils import query_disc
+import lsst.utils.tests
+
+
+class TestAssociationUtils(lsst.utils.tests.TestCase):
+
+    def test_queryDisc(self):
+        """Test that doing a disc query of healpy using our wrapper works as
+        expected.
+
+        Most other calculations are very simple and not worth testing.
+        """
+        # Find nearest pixel to r=45, dec=45.
```

Comment:
Please define the test RA and Dec as variables, and pass those to your functions. Also, please use different numbers for Ra and Dec, so that this test will catch if the order of parameters gets switched. Maybe `raInit = 225` and `decInit = 45`, for example.

Reply:
Okay, this may lead to a change in the number of pixels found in the test just as a warning.